maxentep/maxent.py

```diff
@@ -209,6 +231,18 @@ def __init__(self, restraints, use_cov=False, name='maxent-model', **kwargs):
             self.avg_layer = AvgLayer(self.weight_layer)
         self.lambdas = self.weight_layer.l
         self.prior = prior
+        if ref_traj is not None:
+            self.ref_traj = ref_traj
+            if population_fraction is not None:
```

Comment:
Why is  this indented? Shouldn't it not be affected by ref_traj?

Reply:
No, it should not. If the user inputs ref_traj, the model will display the ref_error metric during the fit. In this case, if no inputs are provided for the population fraction, a uniform population fraction is considered among all the patches for calculating the weighted average trajectory.